gradle/scripts/dependencyDefinitions.gradle

```diff
@@ -123,7 +123,7 @@ ext.externalDependency = [
     "guiceMultibindings": "com.google.inject.extensions:guice-multibindings:4.0",
     "guiceServlet": "com.google.inject.extensions:guice-servlet:4.0",
     "derby": "org.apache.derby:derby:10.12.1.1",
-    "mockito": "org.mockito:mockito-core:4.11.0",
+    "mockito": "org.mockito:mockito-inline:4.11.0",
```

Comment:
Can you provide justification in a comment why we're using inline mockito instead of mockito core? From what I read mockito inline is for experimental features.

Reply:
yeah sure, I'll add a comment for using inline vs core. The rationale was to allow testing of mocked ctors, static and final methods that we are using for iceberg distcp. Moreover, inline mockito transitively brings in the mockito-core as well!